Rakefile

```diff
@@ -3,6 +3,8 @@ require "yast/rake"
 # do nothing in osc:build as it can be builds exclusivelly on s390
 Rake::Task["osc:build"].clear_actions if `uname --machine` !~ /s390/
 
+Yast::Tasks.submit_to((ENV["YAST_SUBMIT"] || :factory).to_sym)
```

Comment:
why needed? if it left empty, it will be easier for everyone, especially when branching. As this is default value.